scripts/run.sh

```diff
@@ -49,6 +58,11 @@ while getopts "p:k:c:t:d:T:h" opt; do
     T)
       TYPE="$OPTARG"
       ;;
+    p)
+      [ "$OPTAGR" != "power-qoriq-p2020-ds" ] && [ "$OPTARG" != "power-mpc5777m-evb" ] && \
```

Comment:
`$OPTAGR` seems dubious to me. Did you mean `"$OPTARG"`?
Can you double-check the whole condition?